uninstall.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -e
+
+uninstall_logid() {
+    if [ 1 -gt ${#1} ]; then
+        printf '%s\n' "No manifest_path detected for uninstall. Skipping..."
+        return
+    fi
+    local manifest_path="$1"
+    echo "manifest_path is $manifest_path"
+    while IFS= read -r line; do 
+        printf '%s\n' "Removing $line.."
+        rm "$line"
```

Comment:
This line needs to be preceded with `sudo` to avoid a "permission denied" error. Or the whole script needs to be run with `sudo` .
It also may need a `-f` to stop an error code from causing this function to exit early, since `set -e` has been set.